specs/workspace_spec.rb

```diff
@@ -0,0 +1,106 @@
+require_relative "test_helper"
+
+describe "Workspace class" do
+  describe "Workspace#initialize" do
+    it "initializes a Workspace object" do
+      VCR.use_cassette("Workspace") do
+        response = Workspace.new
+        expect(response).must_be_instance_of Workspace
+      end
+    end
+  end
+
+  describe "Workspace#select_user" do
+    it "selects a known user by username" do
+      VCR.use_cassette("Workspace") do
+        response = Workspace.new
+        response.select_user("ngocle")
```

Comment:
These values that you are using over and over again would make great constants to have at the top.  That way if you changed slack orgs you could just change a few constants and the tests would work.